src/commands/clear/index.ts

```diff
@@ -0,0 +1,82 @@
+import { SwankyCommand } from "../../lib/swankyCommand.js";
+import { ConfigError, FileError } from "../../lib/errors.js";
+import fs from "fs-extra"
+import path from "node:path"
+import { Args, Flags } from "@oclif/core";
+
+type Folder = {name: string, contractName?: string, path: string}
+
+export default class Clear extends SwankyCommand<typeof Clear> {
+
+  static flags = {
+    all: Flags.boolean({
+      char: "a",
+      description: "Select all the project artifacts for delete"
+    }),
+  };
+
+  static args = {
+    contractName: Args.string({
+      name: "contractName",
+      required: false,
+      description: "Name of the contract artifact to clear",
+    }),
+  };
+
+  async deleteFolder(path: string): Promise<void> {
+    try {
+      await fs.remove(path);
+    } catch (err) {
+      throw new FileError(
+        `Error deleting the folder ${path}.`, {cause: err}
+      );
+    }
+  } 
+
+  public async run(): Promise<any> {
+
+    const { flags, args } = await this.parse(Clear);
+
+    if(args.contractName === undefined && !flags.all) throw new ConfigError("You need to send any flag or argument.")
```

Comment:
The config error message can be more explicit, something like: "Specify a contract name or use the --all flag to delete all artifacts."